recipes/base_install.rb

```diff
@@ -32,10 +32,10 @@
     end
   end
   if node['platform'] == 'centos' && node['platform_version'].to_i == 8
-    # Enable PowerTools Repo so *-devel packages can be installed with DNF
+    # Enable powertools repo so *-devel packages can be installed with DNF
     # Enable EPEL repos
     execute 'dnf enable powertools and EPEL repos' do
-      command "dnf config-manager --set-enabled PowerTools && dnf install -y epel-release"
+      command "dnf config-manager --set-enabled powertools && dnf install -y epel-release"
```

Comment:
This will not work with older minor version.
Also `dnf install -y epel-release` is duplicated, because it's already present in https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/amis/packer_centos8.json#L162

Reply:
Should we keep it here for the case of creating cluster from bare AMI?

Reply:
Fixed